### PR TITLE
Tournament

### PR DESCRIPTION
**Pull request by rrjsheth**
code for progress bar and adding tint ( will use after we add code for carousel)

### CODE REVIEW

File: tournament.js [hunk 1]
```diff
@@ -23,17 +30,24 @@ $(document).ready(function() {
...
 
+	//document.querySelector('#p1').MaterialProgress.setProgress(0);
 	//$('#topCard.mdl-card__title').css("background-size",50+"px " + 70 + "%");
 	//$('#topCard.mdl-card__title').css("background-repeat","no-repeat");
 })
 
 $("#topButton").click(topButtonClick);
 $("#bottomButton").click(bottomButtonClick);
 
+function doNothing() {
```
Comment (cwguan) [inline tournament.js]: If this actually does nothing, can you remove it?

File: restaurantInput.html [hunk 2]
```diff
@@ -32,6 +32,9 @@ <h2 class="mdl-card__title-text  description-title">Enter Your Restaurants (2 mi
                             Start typing your chosen restaurants & select to enter them into the tournament!
                         </div>
                     </div>
+                    <!-- Simple MDL Progress Bar -->
+                <div id="p1" class="mdl-progress mdl-js-progress"></div>
```
Comment (cwguan) [inline restaurantInput.html]: Is there supposed to be a progress bar in restaurantInput? Is it supposed to be empty & represent the start state (like the first round will have some progress when you go to tournament.html)?

File: tournament.js [hunk 3]
```diff
@@ -23,17 +30,24 @@ $(document).ready(function() {
...
+	console.log("nothing");
+	// body...
+}
 function topButtonClick(event){
 	totalList.push(totalList.shift()); //add the first element in html
 	totalList.shift();
-
+	numRounds++;
+	//document.querySelector('#p1').MaterialProgress.setProgress(100-( (restaurantLeft*100.0)/totalLength ));
```
Comment (cwguan) [inline tournament.js]: Remove this comment if that formula doesn't work/we don't want

File: tournament.js (hunk 1 above)
Comment (rrjsheth) [reply tournament.js]: yes

File: restaurantInput.html (hunk 2 above)
Comment (rrjsheth) [reply restaurantInput.html]: there isn't supposed to be

**Pull request merged**